pydocx/models.py

```diff
@@ -142,14 +182,42 @@ def child_handler(child):
             # Save the handler
             child_handlers[tag_name] = create_child_handler(field)
 
-        # Process each child
-        for child in element:
-            # Does this child have a corresponding field?
-            field_name = tag_name_to_field_name.get(child.tag, None)
-            if field_name:
-                # Execute the handler
-                handler = child_handlers.get(child.tag, None)
-                if callable(handler):
-                    kwargs[field_name] = handler(child)
+        # Build a mapping of tag names to collections
+        collection_member_to_collections = defaultdict(list)
+        for field_name, field in collections.items():
+            for tag_name in field.name_to_type_map.keys():
+                collection_member_to_collections[tag_name].append(field_name)
+
+        if element:
```

Comment:
Why do we want this conditional? `for child in element` should do this implicitly, right?